source/comply/audit-log.rst

```diff
@@ -76,8 +257,6 @@ Planned enhancements to logging in general
 
 - Allow discrete logging levels. Currently, an application-wide logging level is configured and any log records matching that level or lower will be emitted. These logging levels will remain, but support for zero or more discrete logging levels will be added, meaning only records matching the current log level or one of the discrete levels are emitted. Within the logging engine, any level below 10 (``trace`` through ``critical``/``fatal``, plus ``reserved``) will behave as it does currently, but any level above 10 will be considered discrete. Audit records will have a level above 10.
 
-- Make logging engine enhancements implemented via the ``mlog`` package and will be compatible with existing usage.
-
 - Allow logging levels and discrete levels to different targets (files, databases, etc) via configuration.
```

Comment:
This also does not apply to auditing, and is in place for general logging and no longer in the planning stage.

Reply:
@wiggin77 Just to confirm you are advocating to remove "- Allow logging levels and discrete levels to different targets (files, databases, etc) via configuration." (line 81) in addition to line 79?

Reply:
All three bullet points above have been implemented and are in product.  Line 77, 79, 81.